pages/accountLists/[accountListId]/tools/fixEmailAddresses/[[...contactId]].page.test.tsx

```diff
@@ -90,7 +95,7 @@ describe('FixEmailAddressesPage', () => {
       expect(queryByTestId('loading')).not.toBeInTheDocument(),
     );
 
-    const contactName = getByText('Test Contact');
+    const contactName = await waitFor(() => getByText('Test Contact'));
```

Comment:
This can be simplified to `findByText`.

```suggestion
    const contactName = await findByText('Test Contact');
```